src/test/functional/steps/cases-heard.ts

```diff
@@ -0,0 +1,95 @@
+import {Then, When} from 'cucumber';
+import * as I from '../utlis/puppeteer.util';
+import {expect} from 'chai';
+import {FunctionalTestHelpers} from '../utlis/helpers';
+
+When('I click the cases heard tab', async () => {
+  const selector = '#tab_cases-heard';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.click(selector);
+});
+
+Then('I can view the areas of law listed', async () => {
+  const elementExist = await I.isElementVisible('#casesHeardForm');
+  expect(elementExist).equal(true);
+});
+
+When('I select area of law with id {int} and {int}', async (areaOfLaw1: number, areaOfLaw2: number) => {
+  const selector1 = '#\\33 4255';
```

Comment:
The ints being passed into this function aren't being used.

Reply:
Made the suggested changes